.gitmodules

```diff
@@ -8,5 +8,5 @@
 	branch = master
 [submodule "ninja"]
 	path = ninja
-	url = git@github.com:rescript-lang/ninja.git
+	url = https://github.com/rescript-lang/ninja
```

Comment:
do you have issue with git protocol?

Reply:
I don't. But I didn't have ssh key configured in the terminal I was using, and updating this submodule failed.

I thought using https would remove one barrier for contributors and I saw other submodules were using it, but can roll this back if needed.

Reply:
Let's maybe keep it as it was, to minimize the diff.

Reply:
done